examples/keycode.ts

```diff
@@ -3,7 +3,7 @@
 import {
   KeyCode,
   parse,
-} from "https://deno.land/x/cliffy@v0.20.1/keycode/mod.ts";
+} from "https://deno.land/x/cliffy@v0.24.3/keycode/mod.ts";
```

Comment:
Would be nice if you could change the imports to use local file imports. Than we don't need to update it on every release.